File: ucloud_joys.php
<?php
$file = fopen("UCloud.txt", "r") or die("Unable to open file UCloud.txt!");
$total = 0;
$i = 0;
while(!feof($file)) {
    $text = fgets($file);
    $count = substr_count($text, "UCanUup");
    echo "Line ".$i." : ".$count."\n";
    $total += $count;
    $i++;
}
echo "Total: ".$total."\n";
fclose($file);
?>

File: ucloud_joys.rb
total = 0
i = 0
File.open('UCloud.txt', 'r').each do |text|
    count = text.split('UCanUup').length - 1
    print "Line #{i} : #{count}\n"
    total += count
    i += 1
end

print "Total: #{total}\n"